### PR TITLE
[283::Show Fee Warning] Address PR comments

### PR DESCRIPTION
**Pull request by W3stside**
Merges into #283 

Addresses PR comments found above in PR #283 

### Emphasis on button if Fee > Input
Screenshot from 2021-03-30 13-57-50

### CODE REVIEW

**Review by anxolin — COMMENTED** [review]
Great! 
I feel we have at least 3 different styles for different error messages that tell you you can't trade. We should make sure our styles are coherent.

File: src/custom/pages/Swap/SwapMod.tsx [hunk 1]
```diff
@@ -456,9 +456,13 @@ export default function Swap() {
               </ButtonPrimary>
             ) : noRoute && userHasSpecifiedInputOutput ? (
               isFeeGreater ? (
-                <GreyCard style={{ textAlign: 'center' }}>
-                  <TYPE.main mb="4px">Fees exceed input amount</TYPE.main>
-                </GreyCard>
+                <RowBetween>
+                  <ButtonError buttonSize={ButtonSize.BIG} error id="swap-button" disabled>
+                    <Text fontSize={20} fontWeight={500}>
```
[the remark this answers is not in the document]
Comment (W3stside) [reply src/custom/pages/Swap/SwapMod.tsx:461]: What do you mean? 

I am just reusing the style Uni does for their errors and the swap button, you can see it in action below this code in `pages/swap`

**Comment by W3stside** [comment]
> Great!
> I feel we have at least 3 different styles for different error messages that tell you you can't trade. We should make sure our styles are coherent.

Yeah i feel that. The reason i went with the red coloured button for Fee error is because of: https://gnosisinc.slack.com/archives/CC3C5SMB5/p1616163283037000?thread_ts=1616163034.036600&cid=CC3C5SMB5

**Review by alfetopito — APPROVED** [review]

**Review by anxolin — COMMENTED** [review]
I'm very curious how this PR will play out with the tooltip that explains the received amount. 

It will nice to explain why the user get's 0 :P

File: src/custom/pages/Swap/SwapMod.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (anxolin) [reply src/custom/pages/Swap/SwapMod.tsx:461]: yes, I'm refering to this thing they do of `fontWeight={500} fontSize={20}` it feels strange to me to mix this CSS styles in the markup

**Review by anxolin — APPROVED** [review]

**Pull request merged**